inc/update.class.php

```diff
@@ -473,6 +473,20 @@ public function doUpdates($current_version = null) {
          case "9.5.0-dev":
             include_once "{$updir}update_94_95.php";
             update94to95();
+
+         case "9.5.0":
+         case "9.5.1":
+         case "9.5.2":
+         case "9.5.3":
+         case "9.5.4":
+         case "9.5.5":
+         case "9.5.6":
+         case "9.5.7":
+         case "9.5.8":
+         case "9.5.9":
+         case "x.x.x-dev":
```

Comment:
```suggestion
         case "x.x.x":
```

Reply:
Nope, this -dev will be used to be able to run migration during development phase when we will replace the placeholder and change `GLPI_VERSION` to use `-dev` suffix.